nixos/modules/services/networking/ddclient.nix

```diff
@@ -75,9 +68,12 @@ with lib;
       };
 
       interval = mkOption {
-        default = 600;
-        type = int;
-        description = "The interval at which to run the check and update.";
+        default = "10min";
+        type = str;
+        description = ''
+          The interval at which to run the check and update.
+          See <command>man 7 systemd.time</command> for the format.
```

Comment:
Instead of
```xml
See <command>man 7 systemd.time</command> for the format.
```
it is nicer to directly reference the document:
```xml
See
<citerefentry>
  <refentrytitle>systemd.time</refentrytitle>
  <manvolnum>7</manvolnum>
</citerefentry>
for the format.
```

Reply:
Nice, thanks!